test/fixtures/session.fixtures.ts

```diff
@@ -9,6 +9,8 @@ import DirectorToSign from 'app/models/session/directorToSign.model'
 import DissolutionConfirmation from 'app/models/session/dissolutionConfirmation.model'
 import DissolutionSession from 'app/models/session/dissolutionSession.model'
 
+export const TOKEN = 'some-token'
```

Comment:
We create this same constant in a lot of unit tests. Thought it'd make sense to add it to a fixture which we can use everywhere